Feat/reports

Sire6715 opened the pull request:
work in progress
having modal issues.

Sire6715 committed:
feat:added report router

Sire6715 committed:
WIP: Report screen

Sire6715 committed:
Merge branches 'main' and 'feat/reports' of https://github.com/AfriSplash-Remotely/AfriSplash-SuperAdmin into feat/reports

Sire6715 committed:
WIP:report modal

sammy-code98 reviewed (changes requested):
@Sire6715 great work man.

i did notice that your design of the report modal is quite different from what  was given on the design here : https://www.figma.com/file/VxoknphZySezGNvfr63r42/Afrisplash-Final-UI?type=design&node-id=10721-89144&t=hj5SVcQn66rmmw9x-0 .

![uche](https://github.com/AfriSplash-Remotely/AfriSplash-SuperAdmin/assets/46542941/19234d3e-f46a-473d-abcc-6f2ab2d9200f)
please compare and make the necessary changes.

Sire6715 commented:
Yes,still yet to work on the UI

Sire6715 committed:
feat:fixed report modal UI

Sire6715 commented:
just completed the report modal.

copy-ninja1 commented on src/Modules/Reports/components/Hero.tsx:
```diff
@@ -0,0 +1,72 @@
+import {
```
Rename this component to Search.tsx

copy-ninja1 commented on src/Modules/Reports/components/Hero.tsx:
```diff
@@ -0,0 +1,72 @@
+import { AiFillCaretDown } from "react-icons/ai";
+
+const Hero = () => {
+  return (
```
this component does not look the same with what is on the ui. please check the ui

copy-ninja1 commented on package.json:
```diff
@@ -9,6 +9,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@chakra-ui/modal": "^2.2.12",
```
remove the package ("@chakra-ui/modal")as it's not been used any were

Sire6715 committed:
fix:renamed Hero component to Search

sammy-code98 committed:
fix: resolve all issues raised by reviewer
~ rename Hero.tsx to Search.tsx
~ fix search component to meet ui design
~ add table footer to reports table
